x-pack/solutions/search/plugins/search_navigation/public/base_classic_navigation_items.tsx

```diff
@@ -55,6 +55,14 @@ export const BaseClassicNavItems: ClassicNavItem[] = [
         },
         id: 'searchApplications',
       },
+      {
```

Comment:
Not sure we should be adding this here. Agents is already in the global nav flyout for classic, so I think we can leave this out of the search navigation IMO.

Reply:
Yeh lets remove this. it introduces odd behaviour where clicking on it will make the elasticsearch classic navigation disappear.